### DISCUSSION TITLE
D3  channel archived on Observable can we change the Slack link on the D3js.org site to the actual d3 slack?

### DISCUSSION THREAD

**Discussion by nharrisanalyst** [d3/d3#3914] [Q&A] [answered]
Since D3 has now been completely removed from the observable slack it only makes sense to have the slack link on the d3js.org website point to the actual d3js slack https://d3js.slack.com/ .

**Comment by curran** [comment]
I love that idea! The original D3 Slack is still going strong.

**Reply by Fil** [reply]
It's not been active for months now, and the archives are lost to time. That is why we want to use the [github discussions](https://github.com/d3/d3/discussions). But we should definitely remove that link!

**Reply by IPWright83** [reply]
I honestly find the D3 Slack (when I do use it) much more useful. I find my things on GitHub just go un-responded to unless other community members see them (which involves watching all the repos).

**Comment by mbostock** [comment, marked as answer]
We don’t recommend Slack.

Slack’s real-time nature places an excessive burden on maintainers and the community to respond immediately; Slack’s design encourages users to pose quick one-off questions rather than having more thoughtful, slower discussions, diminishing its lasting value; Slack’s design further results in frequent repeated questions rather than consolidating discussions into lasting, discoverable, organizable, and votable topics; and Slack being separate from GitHub leads to community and maintainer fragmentation.

If you like Slack, you’re of course welcome to use it, but we think it is detrimental to support and so we’re not recommending it.

We would love it if the community would come to GitHub where discussions can easily be seen and searched by everyone without needing to sign-in to Slack, as well as cross-referenced to the thousands of pre-existing issues.

**Reply by IPWright83** [reply]
It does make a lot of sense, but having tried to contribute it feels like no one watches any of the D3 repositories, while the slack community has been super helpful. 

Not sure if there's a way we can get things more active (other than trying to encourage discussions in GitHub), as it'd be great to have a buzzing community as D3 is such a good library.

**Reply by mbostock** [reply]
If you could encourage folks to ask and answer questions here, that would be appreciated! Part of the reason it’s quiet here is that community support was fragmented into silos — it was overwhelming to check Slack, the Observable Forum, Stack Overflow, Twitter, even the old Google Group, _etc._ Speaking for the maintainers, we’d love to see discussions move here where the development is happening and to have more visibility.

**Reply by IPWright83** [reply]
Is there any way we could get more maintainers involved in D3? It feels like most active development/focus has moved over to Observable/Plot (which is fine!) but I think leaves those of you who are maintainers somewhat stretched. But as someone who's tried to engage it'd be nice to be able to get 👍🏻 or 👎🏻 on improvements etc so we know whether to create custom plugins etc. 

(Sorry if I'm a little biased to action, I got tired of trying to chase up a PR. I'm still really enthusiastic to contribute to the D3 community, but it's tricky when you can't progress things). Still one of my favourite libraries of all time.

**Reply by mbostock** [reply]
My hope is that Observable can, in the future, sponsor additional maintainers to work on D3. Encouraging part-time, unpaid maintainers to contribute is difficult because it takes a lot of time and effort to build trust and shared understanding of D3’s design.

**Comment by nharrisanalyst** [comment]
Ok Thank you for the Answer.

With this being said will the d3.js website be removing the link to slack altogether?
